### PR TITLE
Specifying that Python is exclusively one-liners

### PR DESCRIPTION
**Pull request by zdiemer**
It's important to specify here that your Python work is all one-liners.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ You can usually find me working on...
 * [Pagespeed Insights](https://developers.google.com/speed/pagespeed/insights/)
 * [Chrome UX Report](https://developers.google.com/web/tools/chrome-user-experience-report)
 
-These days I write mostly Go, Javascript, and C++; with some Python & Java in the past.
+These days I write mostly Go, Javascript, and C++; with some Python one-liners & Java in the past.
```
Comment (exterkamp) [inline README.md]: Surely this shouldn't be "some" if it's for one-liners?

```suggestion
These days I write mostly Go, Javascript, and C++; with many Python one-liners & Java in the past.
```

**Review by exterkamp — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ You can usually find me working on...
 * [Pagespeed Insights](https://developers.google.com/speed/pagespeed/insights/)
 * [Chrome UX Report](https://developers.google.com/web/tools/chrome-user-experience-report)
 
-These days I write mostly Go, Javascript, and C++; with some Python & Java in the past.
+These days I write mostly Go, Javascript, and C++; with many Python one-liners & some Java in the past.
```
Comment (exterkamp) [inline README.md]:
```suggestion
These days I write mostly Go, Javascript, and C++; with many Python one-liners & a bit of Java in the past.
```

**Review by exterkamp — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ You can usually find me working on...
 * [Pagespeed Insights](https://developers.google.com/speed/pagespeed/insights/)
 * [Chrome UX Report](https://developers.google.com/web/tools/chrome-user-experience-report)
 
-These days I write mostly Go, Javascript, and C++; with some Python & Java in the past.
+These days I write mostly Go, Javascript, and C++; with many Python one-liners & a bit of Java in the past. I spend my free time playing Kane & Lynch 2.
```
Comment (exterkamp) [inline README.md:8]:
```suggestion
These days I write mostly Go, Javascript, and C++; with many Python one-liners & a bit of Java in the past.
```

**Review by exterkamp — CHANGES_REQUESTED** [review]
This is illegal slander.

**Comment by zdiemer** [comment]
bump @exterkamp

**Comment by exterkamp** [comment]
-1

**Pull request closed without merging**